portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSelectFromListField.testcase

```diff
@@ -230,14 +230,37 @@ definition {
 	}
 
 	@description = "Verify that the Field Reference can be Edited"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test ReferenceCanBeEdited {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131368 ReferenceCanBeEdited pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Select from List",
+			formPageNumber = "0");
+
+		Form.gotoAdvancedTab();
```

Comment:
Hi @leticiachiu , I think this step is missing `3. Add Option 1 as an option for the field.`
Besides that, could you please rebase using the latest master to fix the conflicts? Thanks in advance.